modules/ROOT/pages/xml-sdk.adoc

```diff
@@ -158,6 +158,15 @@ See xref:content-parameters.adoc[more info on roles].
 
 Marks an operation's visibility to either `PUBLIC` (anyone can see and use it) to `PRIVATE` (accessible only for the current module and cannot be seen externally).
 
+For example:
```

Comment:
Per Kim, we should use a sentence fragment like this, so I would just tack the for example onto the end of the previous sentence.